Latest version fails to build

The latest version of this repository does not build:

```
$ go test ./...
package github.com/golangci/gofmt/gofmt
	gofmt/gofmt.go:17:2: use of internal package internal/diff not allowed
```

I believe this problem was introduced in f6ae7c7.

--- a/gofmt/gofmt.go
+++ b/gofmt/gofmt.go
@@ -14,7 +14,6 @@ import (
 	"go/printer"
 	"go/scanner"
 	"go/token"
-	"internal/diff"
 	"io"
 	"io/fs"
 	"math/rand"
@@ -25,6 +24,7 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/golangci/gofmt/gofmt/internal/diff"
 	"golang.org/x/sync/semaphore"
 )
 

